### DISCUSSION TITLE
Element index filter: Change order of options for relationsship fields

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#11550] [Ideas] [answered]
A minor one:

Users nearly always want to filter an element index  by relation to a specific other element, but this option comes last, adding two more to already a lot of required clicks:

![image](https://user-images.githubusercontent.com/60010597/177332773-52e8bf3f-4162-487c-b2f8-20cd1d349b64.png)

Would be nice to have the option 'is related to' at the top.

/Mel

**Comment by brandonkelly** [comment, marked as answer]
Good point! Just released Craft 4.1.1 with this change (7935326d3c52407884cbdd292724380ffaf4c9b1).

**Reply by wsydney76** [reply]
@brandonkelly wow, that was fast, thanks.

However it looks like 'is related to' is now the first option in the list, but 'has a value' is still initially selected:

![image](https://user-images.githubusercontent.com/60010597/177362972-5e083c6c-a164-41e3-ad0d-2858643cd2ca.png)

```html
<select id="condition-conditionRules-1-operator" name="condition[conditionRules][1][operator]" hx-post="http://192.168.0.14:59003/index.php?p=admin/actions/conditions/render&amp;site=de" wtx-context="37DB3BE6-9FB3-40EE-857A-B3252803AF61">                                                    
    <option value="relatedTo">is related to</option>                                                                
    <option value="notempty" selected="">has a value</option>
    <option value="empty">is empty</option>
</select>
```

**Reply by brandonkelly** [reply]
Doh, of course. Fixed that for the next release (dce81d380ef5ca36894024cec3dbb6f4f601d52f).

**Reply by brandonkelly** [reply]
4.1.2 is out with that change.
